src/components/LoginForm..jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {Tab, Tabs, TextField, Stack, Button, Container } from "@mui/material";
+import { Box } from '@mui/system';
+
+const passwordMatchingText = {
+    error: "Passwords do not match",
+    valid: ""
+}
+function LoginForm() {
+    const [currentTabIndex, setCurrentTabIndex] = useState(0);
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [passwordVerification, setPasswordVerification] = useState('');
+    const [passwordMatchingMsg, setPasswordMatchingMsg] = useState(passwordMatchingText.valid)
+    
+    const handleTabChange = (event, tabIndex) => {
+        setCurrentTabIndex(tabIndex);
+        setEmail('')
+        setPassword('')
+        setPasswordVerification('')
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    };
+
+    const handleEmailChange = (event) => {
+        setEmail(event.target.value)
+    }
+
+    const handlePasswordChange = (event) => {
+        setPassword(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
+
+    const handlePasswordVerificationChange = (event) => {
+        setPasswordVerification(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
```

Comment:
I am wondering would a form component library like React Forms been helpful for building this component so we wouldn't have to handle state ourselves?

Reply:
Hey, I actually haven't heard of React Form until now, but doing a quick review, it certainly seems to have a lot of benefits. I can update this to React Forms this week if it's something we want to go forward with. 

- https://firflant.github.io/react-form-component/?path=/docs/introduction-overview--page